Web/Presenters/templates/Group/Edit.xml

```diff
@@ -74,8 +74,9 @@
                             <span class="nobold">{_group_administrators_list}: </span>
                         </td>
                         <td>
-                            <input type="radio" name="administrators_list_display" value="0" {if $club->getAdministratorsListDisplay() == 0}checked{/if}/> {_group_display_only_creator}<br>
-                            <input type="radio" name="administrators_list_display" value="1" {if $club->getAdministratorsListDisplay() == 1}checked{/if}/> {_group_display_all_administrators}<br>
+                            {var isAllAdminsHidden = $club->getManagersCount(true) == 0}
```

Comment:
I'd use `areAllAdminsHidden` since are is plural of is.

Reply:
Done